Products/ZenModel/migrate/Migrate.py

```diff
@@ -414,7 +414,7 @@ def orderedSteps(self):
     def list(self):
         print ' Ver      Name        Description'
         print '-----+---------------+-----------' + '-'*40
-        for s in self.allSteps:
+        for s in self.determineSteps():
```

Comment:
That's my fault, list() always returns the full list regardless of selected steps and that bothered me. I can remove that from this PR.